pkg/hostagent/pods.go

```diff
@@ -270,6 +270,7 @@ func (agent *HostAgent) podChangedLocked(podobj interface{}) {
 		agent.epDeleted(&epUuid)
 		return
 	}
+	agent.cniToPodID[epMetaKey] = epUuid
```

Comment:
Should we be mapping this against something more unique than pod namespace and pod name ? What if I restart the pod, get a new uuid but namespace and name are same ?

Reply:
Namespace.name is unique within the cluster. Kubelet will not create a new CNI interface for a given Namespace.Name combination before deleting the old one.

Reply:
i see, thats fine then.